openshift/templates/visual-verifier/visual-verifier-deploy.bcbank.param

```diff
@@ -20,7 +21,8 @@ CUSTOM_FILES_MOUNT_PATH=/opt/app-root/src/static/
 # DJANGO_SECRET_KEY=[\w]{50}
 DJANGO_DEBUG=False
 VERIFIER_NAME=Corporate Bank Verifier
-OIDC_RP_PROVIDER_ENDPOINT=https://a2a-vc-authn-controller-dev.pathfinder.gov.bc.ca
+OIDC_RP_PROVIDER_ENDPOINT=https://a2a-vc-authn-controller-dev\.pathfinder.gov.bc.ca
+OIDC_RP_PROVIDER_HOST=a2a-vc-authn-controller-dev.pathfinder.gov.bc.ca
```

Comment:
This parameter value is unnecessary, it will be parsed and injected in the secret by the `readHostName` function.
Applies to other parameter files as well.

Reply:
value removed